src/vanilla.ts

```diff
@@ -45,44 +23,42 @@ export type StoreApi<T extends State> = {
   subscribe: Subscribe<T>
   destroy: Destroy
 }
+
+export const $$storeMutators = Symbol('$$storeMutators')
```

Comment:
Nope not internal, this is required if you want to author a middleware. I'll later add a "How to write a middleware" section.

Reply:
Okay, for middleware authors.
Module variables are really hard to deal with for bundling. (zustand doesn't depend on it now.)
Can this be a string? Otherwise, I might consider using `Symbol.for()`.

Reply:
Yep it can be a string. Would you want to make the middleware symbols (like `$$persist`) strings too? That also can be done. (we don't have to worry about uniqueness as `StoreMutators` requires all keys to be unique)

Reply:
okay, let's not use symbols.
i even think we don't need to export the strings, to make sure that it doesn't depend on module variables, at least for some time.